src/http/GraphQLHttpServer.ts

```diff
@@ -84,6 +84,7 @@ export class GraphQLHttpServer<TContext> {
       const response = await this.execute(req);
 
       res.status(response.status).set(response.headers).json(response.body);
+      await next();
```

Comment:
I don't believe `next()` in express needs to be awaited. In fact, express defined the return type of `next()` as `void`

Reply:
Check! This is fixed now.